python/hail/ldMatrix.py

```diff
@@ -87,6 +92,8 @@ def read(path):
         jldm = Env.hail().methods.LDMatrix.read(Env.hc()._jhc, path)
         return LDMatrix(jldm)
 
+    @handle_py4j
+    @write_history('path', is_dir=False)
```

Comment:
Path is a directory if `parallel_write=True`, so with path `/here/is/dir` I think you'll get a new file in the parent directory: `/here/is/dir.history.txt`.  This would also be true to functions like export on key_table. Is this the behavior you intend?

Reply:
Good call. I need to fix this for parallel_write=True.